x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/ilm/TimeSeriesLifecycleActionsIT.java

```diff
@@ -1545,6 +1548,145 @@ public void testHaltAtEndOfPhase() throws Exception {
         assertBusy(() -> assertFalse("expected " + index + " to be deleted by ILM", indexExists(index)));
     }
 
+    public void testSearchableSnapshotAction() throws Exception {
```

Comment:
We could probably put these somewhere else to start splitting this apart similar to #53488 (up to you of course)

Reply:
Yes, totally, I am about to start doing that on master (and will backport here with the appropriate conflict fixes) as I think the scope of this PR is already quite large and some infrastructure needs to be extracted from `TimeSeriesLifecycleActionsIT` to be reused across the various new test classes.